api/client/build.go

```diff
@@ -0,0 +1,118 @@
+package client
+
+import (
+	"fmt"
+	"github.com/appcelerator/amp/api/rpc/build"
+	"golang.org/x/net/context"
+	"strings"
+)
+
+// RegisterProject registers a project through the amplifier proxy to the build service
+func (a *AMP) RegisterProject(repo string) (*build.Project, error) {
+	a.Connect()
+	defer a.Disconnect()
+	client := build.NewAmpBuildClient(a.Conn)
+	ctx := context.Background()
+	split := strings.Split(repo, "/")
+	if len(split) != 2 {
+		return nil, fmt.Errorf("invalid repo %q", split)
+	}
+	owner := split[0]
+	name := split[1]
+	request := build.ProjectRequest{
+		Owner: owner,
+		Name:  name,
+	}
+	return client.CreateProject(ctx, &request)
+}
+
+// RemoveProject removes a project through the amplifier proxy to the build service
+func (a *AMP) RemoveProject(repo string) (*build.Project, error) {
+	a.Connect()
+	defer a.Disconnect()
+	client := build.NewAmpBuildClient(a.Conn)
+	ctx := context.Background()
+	split := strings.Split(repo, "/")
```

Comment:
Being a bit fussy here, but I would prefer to see all of the places where you are doing this standardized in github helper that splits strings for orgs, repos, builds, etc, and returns what you need or the error.